dev-lang/fpc/fpc_bin-2.6.2.recipe

```diff
@@ -38,9 +67,13 @@ SOURCE_DIR="fpc-2.6.2.i386-haiku"
 BUILD()
 {
 	echo "Please put as prefix: $prefix"
+	#give user time to copy prefix
+	sleep 5s
```

Comment:
if the install script reads stdin for input then:
- you should be able to use a heredoc or some pipe to send it anyway,
- it's broken and must be fixed :-D